src/app/globals.css

```diff
@@ -64,6 +63,44 @@ input::-webkit-inner-spin-button {
 	animation: fadeIn ease-in-out 0.3s forwards;
 }
 
+.h-with-navbar {
+	height: calc(100dvh - 56px);
+}
+
+.css-nhh29r-MuiInputBase-root-MuiOutlinedInput-root.Mui-focused .MuiOutlinedInput-notchedOutline,
+.css-wftkto-MuiInputBase-root-MuiOutlinedInput-root-MuiSelect-root.Mui-focused .MuiOutlinedInput-notchedOutline,
+.css-1tfnmf9-MuiInputBase-root-MuiOutlinedInput-root.Mui-focused .MuiOutlinedInput-notchedOutline {
+	border-color: #09090b;
+}
+
+.css-1dsddae-MuiFormLabel-root-MuiInputLabel-root.Mui-focused {
+	color: #09090b;
+}
+
+.css-1j45xve-MuiFormLabel-root.Mui-focused,
```

Comment:
I'm not quite sure of these class names. They seems to be auto-generated by some CSS-in-JS library, so they aren't stable.
Maybe you could use the `sx` prop?